### PR TITLE
Image basics

### PR DESCRIPTION
**Pull request by jcbrockschmidt**

### CODE REVIEW

File: include/img.h [hunk 1]
```diff
@@ -1,3 +1,5 @@
+#include <png.h>
```
Comment (RobertCochran) [inline include/img.h]: Managed to sneak this back in. Oops!

File: src/img.c [hunk 2]
```diff
@@ -1,6 +1,5 @@
 #include <stdio.h>
 #include <stdlib.h>
-#include <png.h>
```
Comment (RobertCochran) [inline src/img.c]: You may want to leave this. It makes it explicitly clear that png.h has been
included.

File: include/img.h (hunk 1 above)
Comment (RobertCochran) [inline include/img.h:1]: img.h still needs an include guard!

**Comment by RobertCochran** [comment]
"Reject RGB images for now"...

> /\* do not process PNGs that are not plain RGB */

Hmm... someone is confused. :)

Do a `git commit --amend` to fix the message. Go ahead and force push if you
need to, you won't hurt anything.

**Comment by jcbrockschmidt** [comment]
How's it look now?

**Comment by RobertCochran** [comment]
Yes

**Pull request merged**